### DISCUSSION TITLE
DNS over TLS don't work

### DISCUSSION THREAD

**Discussion by Actraiser69** [AdguardTeam/AdGuardHome#3695] [Q&A]
Hello,

I use Adguard Home since one year. 
Last week, I have renew my certificat with cerbot.
AdGuard Home work but the private DNS stop to work.

My first installation have been done with Snap, I try to install with the script on the Github and the problem continue.

My certificate is valid, where I can find log ? Can you help me ?

**Comment by iliedragos** [comment]
Same here! I have one self hosted server with AdGuard at my home in Romania, another one at work and one in UK. ALL of them don't work anymore. Certificates are valid. On Android, any Android, any phone and any version from 9 and above, doesn't connect to the servers. It also doesn't matter the ISP (wifi or mobile data) from which I try to connect. All this is happening since last friday. I also updated to the latest beta without success.

P.S. I also tried with Ubuntu instead of Debian with the stable and test builds of AdGuard. No luck. Android is still not connecting to DoT. The only thing that works is through Intra but that is kind of a downside.

Please provide some kind of feedback or help since I've been looking over my head to find a solution ever since.

**Comment by Actraiser69** [comment]
The DNS server is functioning well so the concern is not on the filtering lists. for DNS over TLS, the certificate is valid. If you have the same problem at the same time, it's weird! I don't see where it can come from and I can't find a log.

**Reply by iliedragos** [reply]
Ok. So I went further in investigating this. I bought a VPS server and made a DNS on my main domain. Also bought a certificate and installed it. It seems there's a big issue with letsencrypt and their root certificate which is not recognized by Android devices anymore. So, if you want to fix this issue you need a public hostname for your Adguard, a fixed IP and a certificate. I bought one for 5$ per year. I got rid of all the fuss with letsencrypt and all works well.
Hope this helps.

**Reply by unicast92** [reply]
You can delete the last Cert Part manually in your full chain cert for the DNS over TLS validation. 

Android look first for the X3 cert and second for the X1 cert of let's encrypt. X1 is the newer one.

Now you can use DNS over TLS with the new Let's encrypt cert on Android phones.

**Reply by Actraiser69** [reply]
Hello,
Thanks for your feedback ! i have edit the fullchain.pem file and delete the last block.
DNS over TLS is working again, thank you very much.

**Reply by jackcctse** [reply]
OMG, I spent quiet a few nights searching for a solution and a few more nights for a potential pihole migration! Thank you @Actraiser69 for creating the topic and @unicast92 for the solution.

**Reply by quyleanh** [reply]
You saved me @unicast92
Thank you very much.
